### DISCUSSION TITLE
Is this Project dead?

### DISCUSSION THREAD

**Discussion by bcobra** [microsoft/powercat-creator-kit#448] [General]
Will there be any updates/future development of this the project? For Example, the people picker is still in preview and as far as I know there is no modern people picker control. The modern combo box seems to fall short.
Thanks,
B

**Comment by bcobra** [comment]
I'll take your silence that this project is in fact dead

**Comment by denise-msft** [comment]
@bcobra thank you for your question and valued feedback.

Due to limited resources, our team has announced we are not providing updates on regular cadence and releasing updates to the best of our ability. This week we released an update with several new controls and fixes to some existing ones.

Much of our efforts internally have been focused on contributing to the new modern controls available in product with full engineering team support, with successors of Creator Kit components implemented in the new version 9/ Fluent 2 styling and enhanced capabilities and aesthetics, which we highly encourage customers to move over to or choose for new apps before deciding to take dependency on Creator Kit components.

We do not offer a combo box in the Creator Kit, so there is no successor to compare other than the legacy combo box. Please share feedback on modern controls directly in the studio feedback feature when you select the control so the controls team can track what you need.
